dashboard/dashboard.py

```diff
@@ -19,6 +20,11 @@
     get_existing_subscriber_preferences,
     update_existing_subscriber_info
 )
+from streamlit_graphs.release_type_chart import visualize_release_types
+from streamlit_graphs.sales_by_country import visualize_country_sales
+from streamlit_graphs.sales_over_time import visualize_sales_per_hour
+from streamlit_graphs.top_artist_sales import visualize_sales_per_artist_over_time
+from streamlit_graphs.top_genre_sales import visualize_genre_sales
```

Comment:
Does it make sense to have all these in separate files? We could combine them into a 'streamlit_graphs' script?

Reply:
i think so, as having all the graphs in one script would make it a very long script and difficult to debug a chart